test/util.sh

```diff
@@ -35,6 +35,9 @@ docker() {
         up)
           shift
           command docker compose up --timeout 3 "$@";;
+        down)
+          shift
+          command docker compose down --timeout 3 "$@";;
```

Comment:
I'm curious how this works. If a timeout for the containers is set on `up`, why does `down` need to know of it as well?